### ISSUE TITLE
Unificator deletes some surfaces from the unified mesh.

### ISSUE THREAD

**Issue by nilsschmidt1337** [nilsschmidt1337/ldparteditor#650]
_Philo wrote_:

> Trying to fix a vertex mismatch I discovered a problem with unificator, that deletes some surfaces from the unified mesh (and in an assymetrical way, makes me think of a subtle calculation bug...)
> Attached the original file, result with command line unificator (default unification distance + snap on vertices, threshold 0.2), result with LDPE unificator using the same values (**snap on vertices + subpart vertices**). 

[example-files.zip](https://github.com/nilsschmidt1337/ldparteditor/files/1544779/example-files.zip)

Simplified testcase:

```
0 
0 Name: new.dat
0 Author: Nils Schmidt [BlackBrick89]
0 !LDRAW_ORG Unofficial_Part
0 !LICENSE Redistributable under CCAL version 2.0 : see CAreadme.txt

0 BFC CERTIFY CCW

4 10 -21.5147 -2 -70 -25.7574 -2 -65.7574 -25.6568 -2 -65.6568 -20 -2 -68
4 10 -20 -2 -72 -25.6568 -2 -74.3432 -25.7574 -2 -74.2426 -21.5147 -2 -70
4 10 -18.4853 -2 -70 -20 -2 -72 -21.5147 -2 -70 -20 -2 -68
4 10 -14.2426 -2 -65.7574 -18.4853 -2 -70 -20 -2 -68 -14.3432 -2 -65.6568
4 10 -14.3432 -2 -74.3432 -20 -2 -72 -18.4853 -2 -70 -14.2426 -2 -74.2426
1 10 -30 -2 -70 -4.24264 0 4.24264 0 -1 0 4.24264 0 4.24264 2-4ndis.dat
4 10 -25.6568 -2 -74.3432 -25.7574 -2 -74.2426 -20 -2 -72 -14.3432 -2 -74.3432
1 10 -10 -2 -70 -4.24264 0 -4.24264 0 -1 0 -4.24264 0 4.24264 3-4ndis.dat
```

**Comment by nilsschmidt1337** [comment]
Probably, the threshold of 0.2 LDU is just too high? There is no problem with 0.1 LDU...

**Comment by nilsschmidt1337** [comment]
Closed with commit c1bd8e1.
